apps/nav-veileders-flate/src/components/tiltak/endre-deltakelse-modaler/AvsluttDeltakelseModal.tsx

```diff
@@ -186,6 +266,28 @@ export const AvsluttDeltakelseModal = ({
         <Detail size="small" className="mb-4">
           {getEndrePameldingTekst(pamelding.digitalBruker)}
         </Detail>
+
+        {forslag && sluttdatoFraForslag && sluttaarsakFraForslag && (
+          <Box
+            background="surface-neutral-moderate"
+            padding={{ xs: '2', md: '6' }}
+            borderRadius={{ md: 'large' }}
+            className="mt-4"
+          >
+            <Heading level="6" size="small">
+              Forslag fra arrangør:
+            </Heading>
+            <BodyLong className="mt-2" size="small">
+              Ny sluttdato: {formatDateFromString(sluttdatoFraForslag)}
+            </BodyLong>
+            <BodyLong className="mt-2" size="small">
+              Årsak til avslutning:{' '}
+              {getForslagEndringAarsakText(sluttaarsakFraForslag)}
+            </BodyLong>
+            <BodyLong size="small">Begrunnelse: {forslag.begrunnelse}</BodyLong>
+          </Box>
```

Comment:
Jeg tenker at dette også bør være en egen komponent. Ellers blir det fort litt vanskelig å gjøre endringer, spesielt de visuelle.

Reply:
Det var lurt. Det ble mye ryddigere også!

Reply:
Ser ut som at jeg kan gjenbruke selve detaljene som er spesifikke for hver forslagstype (hvertfall sånn skissene er nå), og så måtte jeg lage egen komponent for den generelle modalvisningen av forslagene.